src/serviceWorker.tsx

```diff
@@ -83,17 +83,15 @@ const registerValidSW = (swUrl: string, config?: Config) => {
               if (config && config.onUpdate) {
                 config.onUpdate(registration);
               }
-            } else {
+            } else if (config && config.onSuccess) {
               // At this point, everything has been precached.
               // It's the perfect time to display a
               // "Content is cached for offline use." message.
 
               console.info("Content is cached for offline use.");
 
               // Execute callback
-              if (config && config.onSuccess) {
-                config.onSuccess(registration);
-              }
+              config.onSuccess(registration);
             }
           }
```

Comment:
note: this refactor isn't functionally identical

Reply:
Right..

Reply:
Reverted